mod/metrics/lib/card/all_answer_query.rb

```diff
@@ -1,8 +1,9 @@
 class Card
   class AllAnswerQuery
-    def initialize filter, paging
+    def initialize filter, sorting, paging
```

Comment:
this really just wasn't handling sorting at all before.  Just hard-coded to "name"